dislib/cluster/daura/base.py

```diff
@@ -0,0 +1,106 @@
+from itertools import chain
+
+from pycompss.api.api import compss_wait_on
+from pycompss.api.parameter import COLLECTION_IN, INOUT
+from pycompss.api.task import task
+from sklearn.base import BaseEstimator
+
+import numpy as np
+
+
+class Daura(BaseEstimator):
+
+    def __init__(self, cutoff):
+        """ Perform Daura clustering.
+
+        See:
+        lockhartlab.squarespace.com/blog/2018/1/14/clustering-with-daura-et-al
+
+        Parameters
+        ----------
+        cutoff : float
+            Distance to determine the neighbors of a sample.
+        """
```

Comment:
This should go below the class declaration.

Use a References subsection for the reference (see C-SVM for example)

Reply:
Done. The link wasn't a reference, just an info webpage, but I have formatted it. And I have added the reference to the original paper.